rewrite-maven/src/main/java/org/openrewrite/maven/AddCommentToTag.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Incubating;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.marker.Markers;
+import org.openrewrite.xml.XPathMatcher;
+import org.openrewrite.xml.tree.Content;
+import org.openrewrite.xml.tree.Xml;
+
+import java.util.*;
+
+import static org.openrewrite.Tree.randomId;
+
+@Incubating(since = "7.24.0")
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class AddCommentToTag extends Recipe {
+
+    @Option(displayName = "XPath",
+            description = "An XPath expression used to find matching tags.",
+            example = "/project/dependencies/dependency")
+    String xPath;
```

Comment:
`xPath` parameter would be useful to be able to differentiate between adding a comment in managed dependencies and "normal" dependencies, right? since the recipe is totally oriented to TAGs of type dependency...

Reply:
> `xPath` parameter would be useful to be able to differentiate between adding a comment in managed dependencies and "normal" dependencies, right? since the recipe is totally oriented to TAGs of type dependency...

Yes, the intent of `xPath` is to enable the ability to distinguish which group of tags the recipe will add a comment to.